lib/renderer/renderer-webpack.js

```diff
@@ -244,25 +282,25 @@ class Renderer {
      * @returns {Promise<{client: string, server: string, clientBundlePath: string}>}
      */
     async _makeBundle(config, filePath) {
-        const parsedBundlePath = path.parse(filePath);
-        const bundlesDir = path.join("/expressvue", "bundles");
-        const clientBundleFile = path.join(bundlesDir, `${parsedBundlePath.base}.client.js`);
-        const serverBundleFile = path.join(bundlesDir, `${parsedBundlePath.base}.server.js`);
-        const vm = this;
-        //file was not found so make it
-        await this._webPackCompile(config.config, filePath);
-        const bundleFilePaths = find.fileSync(/\.js$/, this.bundlePath);
-        await this._loadBundleFilesToMemory(bundleFilePaths);
-        let serverBundle = vm.mfs.readFileSync(serverBundleFile, "utf-8");
-        let clientBundle = vm.mfs.readFileSync(clientBundleFile, "utf-8");
-        if (!serverBundle || !clientBundle) {
+        try {
+            const bundlePath = this.getBundleFilePath(filePath);
+            //file was not found so make it
+            await this._webPackCompile(config.config, filePath);
+            const bundleFilePaths = find.fileSync(/bundle\.js$/, this.expressVueFolder);
+            await this._loadBundleFilesToMemory(bundleFilePaths);
+            let serverBundle = this.mfs.readFileSync(bundlePath.memory.filename.server, "utf-8");
+            let clientBundle = this.mfs.readFileSync(bundlePath.memory.filename.client, "utf-8");
```

Comment:
use `const` instead of `let` for `serverBundle` and `clientBundle`. Also use async version if there is on and put them in a `Promise.all` together with `this._loadBundleFilesToMemory(bundleFilePaths)` possibly